core/templates/dev/head/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.html

```diff
@@ -56,7 +56,7 @@
     </button>
   </div>
   <div class="oppia-content-wrapper">
-    <button class="btn oppia-audio-button oppia-audio-bar-button-transition protractor-test-accessibility-translation-upload-audio protractor-test-upload-audio-button"
+    <button class="btn oppia-audio-button oppia-audio-bar-button-transition protractor-test-upload-audio protractor-test-accessibility-translation-upload-audio"
```

Comment:
Why this change? We actually do suffix '-button' elsewhere in the codebase, so I'd suggest add it back here and add it to the other protractor classes that are added.

Reply:
Fixed in 6993eb8c7